hooks/pre-commit.bash

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 echo “------------ Running pre-commit hook------------”
-xcodebuild -quiet -workspace RoutesApp_ios.xcworkspace -scheme "RoutesApp_ios" -destination 'platform=iOS Simulator,name=iPhone 13 Pro,OS=15.5' test
+xcodebuild ARCHS=x86_64 ONLY_ACTIVE_ARCH=NO -quiet -workspace RoutesApp_ios.xcworkspace -scheme "RoutesApp_ios" -destination 'platform=iOS Simulator,name=iPhone 13 Pro,OS=15.5' test
```

Comment:
Does this change affect something in the hook for testing?. If so, notify the team to update theirs